README.md

```diff
@@ -195,6 +202,31 @@ class CustomSolrSearchIndex extends SolrSearchIndex
 
 After setting up your SolrSearchIndex, run `sake dev/tasks/Solr_Configure`.
 
+## Building the elemental frontend assets
+
+This module uses the [SilverStripe Webpack module](https://github.com/silverstripe/webpack-config), and inherits
+things from the core SilverStripe 4 modules, such as a core variable sheet and Javascript components.
+
+When making changes to either the SASS or Javascript files, ensure you change the source files in `client/src/`.
+
+You can have [yarn](https://yarnpkg.com/en/) watch and rebuild delta changes as you make them (for development only):
+
+```
+yarn watch
+```
+
+When you're ready to make a pull request you can rebuild them, which will also minify everything. Note that `watch`
+will generate source map files which you shouldn't commit in for your final pull request. To minify and package:
```

Comment:
Is it possible for us to preemptively  `.gitignore` them? e.g. `dist/**/*.map` or whatever they come out as?

Reply:
Good idea